server/helpers.js

```diff
@@ -119,3 +121,13 @@ export function isCredentialError(err) {
            err.errCode === getErrorCode('INVALID_PARAMETERS') ||
            err.errCode === getErrorCode('NO_PASSWORD');
 }
+
+export function setupMoment(locale) {
+    if (locale) {
```

Comment:
Other idea: rather than leaking the `moment` value, could we have an exported function here, `formatDateToLocaleString(date) { return moment(date).format('L'); }`? This would hide the implementation detail that is `moment`, and in particular the `'L'` constant that is in several files.